package.json

```diff
@@ -19,10 +19,11 @@
     "build": "echo yes | npm run release && cp release/woocommerce-services.zip ./",
     "up": "docker-compose up --build --force-recreate -d && ./bin/docker-setup.sh",
     "down": "docker-compose down",
-    "test:e2e-docker-up": "npx wc-e2e docker:up",
+    "test:e2e-docker-up": "env WP_VERSION=6.4.1 TRAVIS_MARIADB_VERSION=10.8.3 npx wc-e2e docker:up",
     "test:e2e-docker-down": "npx wc-e2e docker:down",
     "test:e2e-docker-ping": "bash ./tests/bin/wait-for-e2e-build.sh",
-    "test:e2e": "npx wc-e2e test:e2e",
+    "test:e2e": "WC_E2E_SCREENSHOTS=1 npx wc-e2e test:e2e",
```

Comment:
Will this upload the screenshots somewhere? If so that would be great to help determine what causes the failure.

Reply:
ah, that's a left over, I'd remove it and we can have an issue to upload them somewhere as they are very handy for e2e tests.